Source/EasyNetQ/Producer/IResponderConfiguration.cs

```diff
@@ -27,5 +43,17 @@ public IResponderConfiguration WithQueueName(string queueName)
             QueueName = queueName;
             return this;
         }
+
+        public IResponderConfiguration WithDurable(bool durable = true)
+        {
+            Durable = durable;
+            return this;
+        }
+
+        public IResponderConfiguration WithExpires(int? expires)
```

Comment:
Does it make sense to make expires parameter nullable?

Reply:
At first I thought to make it non nullable but then it will not be possible to reset the property to the default value. I looked at my code again and it really uses non nullable. Done.